### DISCUSSION TITLE
Average Heart Rate

### DISCUSSION THREAD

**Discussion by tituscmd** [InfiniTimeOrg/InfiniLink#133] [Ideas]
In the menu where you can check your heart rate, it would be nice to have a little section where it tells you your average heart rate in that given data range (depending on what your slider is set to).

**Comment by liamcharger** [comment]
Just made a commit that adds this feature!

**Reply by tituscmd** [reply]
Awesome! Looking forward to checking it out soon.

**Comment by tituscmd** [comment]
I’ve been thinking about this some more. Would it be smarter to use the median of the selected data? The watch occasionally gives extreme readings, like 40 BPM while biking, and the median would prevent these outliers from affecting our target value.
